### PR TITLE
UI layout

### LINKED ISSUE #436: UI components for filtering
Checkboxes for the filters:
- Location: America, Europa, Asia, Africa
- Type: Camp, Unconference, Hackathon

And the connection with the logic developed in #435

### LINKED ISSUE #438: UI Layout
Desktop layout for the event index page and for the individual event

### PR DESCRIPTION
**Pull request by marinaaisa**
Closes #436
Closes #438 

This is what is done so far:
Captura de pantalla 2020-03-05 a las 14 53 43

The main thing done in this PR has been the custom tokens for components which it's fixed already.

What is left:
- [x] Fix checkboxes, they can't be marked at the moment and they should be marked.

I will not have time to finish it this issue, so please, assign yourself what is left from this issue.

### CODE REVIEW

**Comment by y4izus** [comment]
This PR is only related to #438? I'm asking this because this issue was thought to build just the layout for both the index events page and the individual event page. You're also developing the filter part (issues #435 and #436)

We can create another issue for the individual event page and apply the changes requested by @delapuente in order to merge the index page. What do you think @marinaaisa , @delapuente ??

Also, if the filters are working, we should have in mind to close issues #435 and #436

**Comment by delapuente** [comment]
Indeed, this includes #436, although #435 is not implemented yet. Filters are dynamically generated but they are not filtering anything yet. Nice catch, @y4izus!

File: pages/events/index.vue [hunk 1]
```diff
@@ -11,23 +12,74 @@
...
 import EventCard from '~/components/cards/EventCard.vue'
 
+import {
+  ORDERED_LOCATION_FILTERS,
+  ORDERED_TYPE_FILTERS,
+  Filter
+} from '~/constants/filters'
+
 @Component({
   layout: 'event',
```
[the remark this answers is not in the document]
Comment (delapuente) [reply pages/events/index.vue]: It is fine if, right not, we simply solve the problem of not having the same menu and footer than the rest of the web.

File: constants/filters.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  asia: Filter
+}
+
+type TypesFilter = {
+  camp: Filter,
+  unconference: Filter,
+  hackathon: Filter
+}
+
+const LOCATION_FILTERS: LocationFilter = {
```
[the remark this answers is not in the document]
Comment (marinaaisa) [reply constants/filters.ts]: Done!

File: pages/events/index.vue [hunk 3]
```diff
@@ -11,23 +12,74 @@
...
+        <div class="event-page__filters-checkboxes">
+          <fieldset class="bx--fieldset">
+            <legend class="bx--label">
+              Location
+            </legend>
+            <cv-checkbox
+              v-for="location in locations"
+              :key="location.value"
+              v-model="locationModel"
+              :name="location.value"
```
[the remark this answers is not in the document]
Comment (marinaaisa) [reply pages/events/index.vue]: I did it but it still not working :(

File: layouts/event.vue [hunk 4]
```diff
@@ -4,10 +4,7 @@
 
 <style lang="scss">
 @import '~/assets/scss/theme.scss';
+@import '~/assets/scss/custom-carbon.scss';
```
[the remark this answers is not in the document]
Comment (marinaaisa) [reply layouts/event.vue]: Could you do it for me, please? I don't think I have time to do it anymore.

File: pages/events/index.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (marinaaisa) [reply pages/events/index.vue]: I added the header and footer as we spoke this morning. I hope this is okay now.

File: pages/events/index.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (delapuente) [reply pages/events/index.vue]: You're right. I removed the name and, after the hot-reload, it worked. In the same way, navigating out and back the event index worked. This is related to the fact that there is no interactivity in the prerendered version of the page. The tabs and filters are _client-only_ components and should be inside the Nuxt [`<client-only>` component](https://nuxtjs.org/api/components-client-only/).

**Pull request merged**